lib/kennel/tags_validation.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+module Kennel
+  module TagsValidation
+    def validate_json(data)
+      super
+
+      # ideally we'd avoid duplicated tags, but that happens regularly when importing existing monitors
+      data[:tags] = data[:tags].uniq
+
+      # keep tags clean (TODO: reduce this list)
+      bad_tags = data[:tags].grep(/[^A-Za-z:_0-9.\/*@!#-]/)
```

Comment:
Why that set of characters?

Reply:
set that we currently need ... ideally it would be less